test/src/examples_complete_test.go

```diff
@@ -1,78 +1,86 @@
 package test
```

Comment:
Please also add a test for enabled=false

See these references

https://github.com/cloudposse/terraform-aws-api-gateway/blob/d0627f81dac5354fdf1621256d6161617676ac46/test/src/common.go#L13

https://github.com/cloudposse/terraform-aws-api-gateway/blob/d0627f81dac5354fdf1621256d6161617676ac46/test/src/examples_rest_api_test.go#L67-L69